kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/CreateTopicRequest.java

```diff
@@ -72,6 +83,9 @@ static CreateTopicRequest fromJson(
 
     public abstract Builder setReplicationFactor(@Nullable Short replicationFactor);
 
+    public abstract Builder setReplicasAssignments(
+        @Nonnull Map<Integer, List<Integer>> replicasAssignments);
```

Comment:
We don't use `@Nonnull`. Everything that is not explicitly marked as `@Nullable` in the REST Proxy repo is to be assumed to be non-null.

Reply:
Fixed.